man/man1/oci-kvm.1

```diff
@@ -59,6 +77,19 @@ The
 .B oci-kvm create-pool
 subcommand creates a storage pool to host guest virtual disks. Pools can be created upon local disk, block storage of FSS space.
 
+The
+.B oci-kvm create-network
+subcommand creates a libvirt network on top of OCI vNIC. The created network offers an IP range usable by other KVM guests.
+See the '--virtual-network' option of the
+.B create
+command
+
+The
+.B oci-kvm delete-network
+subcommand deletes a libvirt network previously created by
```

Comment:
subcommand deletes a libvirt network previously created by the

Reply:
done